tests/Functional/ContentProjectRenderTest.php

```diff
@@ -177,6 +205,122 @@ public function testProjectTopGroupRender(): void {
     $file_download_link = $file_row->find('css', '.ecl-file__download');
     $this->assertContains('/test.pdf', $file_download_link->getAttribute('href'));
     $this->assertContains('Download', $file_download_link->getText());
+
+    // Assert bottom region - Stakeholders.
+    $project_stakeholder = $this->assertSession()->elementExists('css', 'div#project-stakeholder');
+
+    // Assert header.
+    $stakeholder_headers = $project_stakeholder->findAll('css', 'h2');
+    $this->assertEqual($stakeholder_headers[0]->getText(), 'Stakeholders');
+
+    // Assert Coordinators field.
+    $stakeholder_sub_headers = $project_stakeholder->findAll('css', 'h3');
+    $this->assertCount(1, $stakeholder_sub_headers);
+    $this->assertEqual($stakeholder_sub_headers[0]->getText(), 'Coordinators');
+    $this->assertOrganisation($project_stakeholder, 'coordinator');
+
+    // Unpublish Coordinator and publish Participant organisations.
+    $coordinator_organisation->set('status', 0);
+    $coordinator_organisation->save();
+    $participant_organisation->set('status', 1);
+    $participant_organisation->save();
+
+    // Reload the page.
+    $this->drupalGet($node->toUrl());
+
+    // Assert Participants field.
+    $project_stakeholder = $this->assertSession()->elementExists('css', 'div#project-stakeholder');
+    $stakeholder_sub_headers = $project_stakeholder->findAll('css', 'h3');
+    $this->assertCount(1, $stakeholder_sub_headers);
+    $this->assertEqual($stakeholder_sub_headers[0]->getText(), 'Participants');
+    $this->assertOrganisation($project_stakeholder, 'participant');
+  }
+
+  /**
+   * Creates organisation entity.
+   *
+   * @var string $name
+   *   Name of the entity. Is used as a parameter for test data.
+   * @var int $status
+   *   Entity status. 1 - published, 0 - unpublished.
+   *
+   * @return \Drupal\oe_content_entity_organisation\Entity\OrganisationInterface
+   *   Organisation entity.
+   */
+  protected function getOrganisationEntity($name, $status): OrganisationInterface {
```

Comment:
@nagyad yad, let's extent OrganisationInterface and add there:
```
interface OrganisationInterface extends CorporateEntityInterface {

  /**
   * Denotes that the organisation is not published.
   */
  const NOT_PUBLISHED = 0;

  /**
   * Denotes that the organisation is published.
   */
  const PUBLISHED = 1;
}
```
then we can invoke createStakeholderOrganisationEntity('name', OrganisationInterface::PUBLISHED). Is it appropriate solution?

Reply:
No. Why would we do that? Simply map the string with the value in the method.

Reply:
Using constants to determine status of the entity is a pattern that is used in NodeInterface or CommentInterface. Such approach
* defines status clearly
* increases readability of the code
* can be reused in different places.

From my point of view mapping for the status increases function complexity and can be useful just in one particular case.

Reply:
The things you wrote are true, but at the moment it does not serve any functionality. However, we could consider to have this in the CorporateEntityInterface rather than on organisation as this is true for all corporate entities.

Reply:
We can put them on `CorporateEntityInterface`.